tramp-hdfs-tests.el

```diff
@@ -14,13 +14,15 @@
 
 ;;; Commentary:
 ;; These are some tests for tramp-hdfs.
+;; "node-1" should be mapped to running HDFS instance via /etc/hosts.
```

Comment:
Can you also put this in running tests section of README ?

Reply:
sure